src/js/components/Meter/stories/MultipleValues.js

```diff
@@ -1,57 +1,54 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import { storiesOf } from '@storybook/react';
 
 import { Grommet, Box, Meter, Stack, Text } from 'grommet';
 import { grommet } from 'grommet/themes';
 
-class MultipleValues extends Component {
-  state = { total: 100 };
+function MultipleValues() {
+  const [ total ] = useState(100);
```

Comment:
why does `total` need to use state rather than just be a const?

Reply:
You are right, it should just be const, unsure why it was state in the first place